saleor/product/models.py

```diff
@@ -207,6 +236,14 @@ def __repr__(self):
     def __str__(self):
         return self.name
 
+    def save(
+        self, force_insert=False, force_update=False, using=None, update_fields=None
```

Comment:
Why not `*args, **kwargs`?

Reply:
Because this is the real signature and when you are using/overriding this method you know exactly what arguments it requires.